src/core/common/date.ts

```diff
@@ -0,0 +1,20 @@
+export function createDateFormatter(
+  locales: string[] | undefined,
```

Comment:
suggestion: because technically `Intl.DateTimeFormat` can accept `string | string[] | undefined`, modifying this type will allow the other uses of it (like in `src/core/server/services/users/download/download.ts`) to just pass the `string` instead.

Same thing with the `formatDate` function here (that's used in `src/core/server/services/users/users.ts`).